src/main/java/org/trips/service_framework/utils/CmsUtils.java

```diff
@@ -0,0 +1,47 @@
+package org.trips.service_framework.utils;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.util.CollectionUtils;
+import org.trips.service_framework.constants.CmsConstants;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+
+public class CmsUtils {
+
+    public static String sanitizeInput(String attribute, String value) {
+        if (Objects.isNull(value) || value.isEmpty() || value.equals(CmsConstants.NOT_APPLICABLE)) {
```

Comment:
1. better to use StringUtils.isEmpty instead of null and blank check explicitly.
2. CmsConstants.NOT_APPLICABLE.equals(value)